src/main/java/edu/kis/powp/jobs2d/command/DriverCommandCallCounterVisitor.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.Iterator;
+
+
+/**
+ * Visitor class for driver commands.
+ */
+public class DriverCommandCallCounterVisitor implements Visitor {
+	private int counter = 0;
+
+	public int getCounter(){
+		return this.counter;
+	}
+
+	@Override
+	public void visit(DriverCommand driverCommand) {

```

Comment:
IMPORTANT: Visit method should not be declared for the top of the hierarchy, but for particular commands and compound command. 
This is crucial for implementing visitors. The single call to visit (using accept) should trigger calculation of correct counter value.